fixtures/cloudflare-dev-module-resolution/src/slash-create.ts

```diff
@@ -0,0 +1,22 @@
+// Note: we can import from the /web entrypoint, if we import from the default one things don't work
+//       both because of missing built-in node modules and also because of other module resolution
+//       issues.
+//       Note that in the readme itself: https://github.com/Snazzah/slash-create?tab=readme-ov-file#using-webservers
+//       they say to use /web, so I do think that this is good enough (should the standard import also work?)
```

Comment:
Maybe this is something to test in the future when we've added node compat support.

Reply:
Sure 🙂👍